src/components/Weather.tsx

```diff
@@ -0,0 +1,43 @@
+import { useWeatherQuery } from '../store/api';
+
+export default function Weather() {
+  const { data: weather, isSuccess } = useWeatherQuery(undefined, {
+    pollingInterval: 600000, // 10 * 60 * 1000, 10 minutes
+  });
+
+  if (!isSuccess) return <></>;
+
+  const iconMap = new Map([
+    ['mist', 'mist'],
+    ['buien', 'rainy'],
+    ['regen', 'rainy'],
+    ['sneeuw', 'rainy'],
+    ['bewolkt', 'rainy'],
+    ['onbewolkt', 'sunny'],
+    ['nachtmist', 'bedtime'],
+    ['hagel', 'weather_hail'],
+    ['wolkennacht', 'bedtime'],
+    ['bliksem', 'thunderstorm'],
+    ['helderenacht', 'bedtime'],
+    ['nachtbewolkt', 'bedtime'],
+    ['halfbewolkt_regen', 'rainy'],
+    ['zwaarbewolkt', 'filter_drama'],
+    ['halfbewolkt', 'partly_cloudy_day'],
+    ['lichtbewolkt', 'partly_cloudy_day'],
+  ]);
```

Comment:
I'd rather see this be defined outside of the component, since now this map is created anew at every render of the page. It won't hurt performance at all, but it's good practice to move it outside of the component.